database/migrate_lexicon_table.js

```diff
@@ -0,0 +1,101 @@
+export function migrate_lexicon_table(tbta_db, project, targets_db) {
+	const transformed_data = transform_tbta_data(tbta_db, project)
+
+	create_tabitha_table(targets_db)
+
+	load_data(targets_db, transformed_data)
+}
+
+/** @param {import('bun:sqlite').Database} tbta_db */
+function transform_tbta_data(tbta_db, project) {
+	const table_names = [
+		'Adjectives',
+		'Adpositions',
+		'Adverbs',
+		'Nouns',
+		'Verbs',
+		'Conjunctions',
+		'Particles',
+		'Pronouns',
+	]
+	const transformed_data = transform_data(table_names)
+
+	return transformed_data
+
+	function transform_data(table_names) {
+		console.log(`Transforming part of speech data from ${tbta_db.filename}...`)
+
+		const transformed_data = table_names.map(table_name => {
+			const singular_part_of_speech = table_name.slice(0, -1)
+
+			const sql = `
+				SELECT 	ID as id,
+							'${project}' as project,
+							Roots as stem,
+							'${singular_part_of_speech}' as part_of_speech,
+							Glosses as gloss,
+							Features as features,
+							Constituents as constituents
+				FROM ${table_name}
+				ORDER BY ID
+			`
+			return tbta_db.query(sql).all()
+		}).flat() // flattens data from each parts of speech table into a single array
+
+		console.log('done.')
+
+		return transformed_data
+	}
+}
+
+/** @param {import('bun:sqlite').Database} targets_db */
+function create_tabitha_table(targets_db) {
+	create()
+
+	empty_existing_table()
+
+	return targets_db
+
+	function create() {
+		console.log(`Creating Lexicon table in ${targets_db.filename}...`)
+
+		targets_db.query(`
+			CREATE TABLE IF NOT EXISTS Lexicon (
+				id					INTEGER,
+				project			TEXT,
+				stem				TEXT,
+				part_of_speech	TEXT,
+				gloss				TEXT,
+				features			TEXT,
+				constituents	TEXT,
+				forms				TEXT
+			)
+		`).run()
+
+		console.log('done.')
+	}
+
+	function empty_existing_table() {
+		console.log(`Cleaning existing data in Lexicon...`)
+
+		targets_db.query(`DELETE FROM Lexicon`).run()
```

Comment:
We should probably only delete the entries for the project that we're migrating, since we're importing data for one project at a time. Same goes for all the other migration scripts.

Reply:
this needs more thought... I don't think this is the PR to address that in actually.

Reply:
will fully address this in #8 but got rid of this buggy code for now:  [07beabb](https://github.com/presciencelabs/tabitha-targets/pull/7/commits/07beabb17688f42d0a0cff4d3ec625b760a9c9e3)